docs/quickstart.rst

```diff
@@ -271,5 +271,50 @@ the Managed Services Tooling bootstrap image.
    If you used the parameters from the example above, the created ISO image
    will be placed in ``~/build/ms-centos-7.iso``
 
-#. Deploy an OSS Tooling appliance using the created ISO image.
+.. _qs-automated-deployment:
 
+Automated deployment of the OSS infrastructure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The deployment of OSS Tools requires at least three hardware or virtual nodes
+which are connected in the same network. These nodes have to be installed from
+the pre-built OSS Tooling ISO.
+
+.. note::
+    During the installation from ISO the `mirantis` user will be created with
+    the password `mirantis`.
+
+After installation of nodes `automation` have to be copied on one of the nodes:
```

Comment:
Currently, we are using the separate machine for deployment. The case with deployment from the first machine was not nested